topics/ui/controls/button.topic

```diff
@@ -350,7 +350,7 @@
             </table>
         </chapter>
         <chapter title="Cancel" id="cancel">
-            <table style="none" column-width="fixed">
+            <table style="none" border="false" column-width="fixed">
```

Comment:
Could you please stick to the same order in elements? It will be easier to scan if any edits are needed.
I mean that the `border` attribute in lines 316, 335, 389 are last, but in 353 and 416 are not last.